packages/design-system/src/components/contextMenu/mask.js

```diff
@@ -64,6 +64,7 @@ export default function Mask({ onDismiss }) {
         data-testid="context-menu-mask"
         hasOnDismiss={Boolean(onDismiss)}
         onClick={onDismiss || noop}
+        onMouseDown={onDismiss || noop}
```

Comment:
🤔 using both onClick and onMouseDown here duplicates the `onDismiss` getting fired. Since mouseDown occurs before the click action takes hold. Do we need both? Could onClick get removed in favor of the mouseDown? If not, then should mouseDown just stop bubbling and never trigger `onDismiss`?

Reply:
Nice catch!